econml/solutions/causal_analysis/_causal_analysis.py

```diff
@@ -8,10 +8,11 @@
 
 import joblib
 import lightgbm as lgb
+from numba.core.utils import erase_traceback
 import numpy as np
 from numpy.lib.function_base import iterable
 import pandas as pd
-from sklearn.base import TransformerMixin
+from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.compose import ColumnTransformer
 from sklearn.ensemble import GradientBoostingClassifier, RandomForestClassifier, RandomForestRegressor
 from sklearn.linear_model import Lasso, LassoCV, LogisticRegression, LogisticRegressionCV
```

Comment:
_CAT_LIMIT seems undefined!

Reply:
The final logistic regression model that we return in the case of "linear", does not have `max_iter=1000`

Reply:
> _CAT_LIMIT seems undefined!

It is: https://github.com/microsoft/EconML/blob/80ea4d5d48a04b16c651624725a263cf327ff8c7/econml/solutions/causal_analysis/_causal_analysis.py#L492-L494

Reply:
> The final logistic regression model that we return in the case of "linear", does not have `max_iter=1000`

Fixed in next push.